src/Og.php

```diff
@@ -269,6 +269,34 @@ public static function isGroupContent($entity_type_id, $bundle_id) {
     return \Drupal::service('og.group_audience_helper')->hasGroupAudienceField($entity_type_id, $bundle_id);
   }
 
+  /**
+   * Gets the Group entities where an entity belongs.
+   *
+   * This works by checking if the bundle has at least one group audience field
+   * filled.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity object.
+   *
+   * @return array
+   *   List of groups where the entity belongs.
+   */
+  public static function getEntityGroups(EntityInterface $entity) {
+    $can_be_group_content = \Drupal::service('og.group_audience_helper')->hasGroupAudienceField($entity->getEntityTypeId(), $entity->bundle());
```

Comment:
Could you save the service into a variable, so you don't need to invoke `Drupal::service` twice?

Reply:
Those functions are static, so they are invoked without the object being initialized. We could not start the service on the constructor. So I think this is the best solution that comes to my mind.

Reply:
no, I believe he means 

```php
$audience_helper = \Drupal::service('og.group_audience_helper');
$audience_fields = audience_helper->getAllGroupAudienc ...
```